packages/components/organisms/f-footer/src/components/FeedbackBlock.vue

```diff
@@ -1,7 +1,10 @@
 // .is-invisible class on data-gtm-feedback element is dictated by gtm script for Usabilla
 <template>
     <div
-        :class="[$style['c-feedback'], 'is-invisible']"
+        :class="[
+            $style['c-feedback'],
+            $style['is-invisible']
```

Comment:
[`is-invisible` is a class from Fozzie](https://github.com/justeat/fozzie/blob/99280c21e6c9c8baa54604ae2c670fd7583db4c8/src/scss/trumps/_utilities.scss#L81) and so shouldn't be reference using the object style otherwise it'll have no effect.